### PR TITLE
Add missing include in AColor.cpp

### LINKED ISSUE #662: Error: 'copy_n' is not a member of 'std' (gcc 8.3.0)
### Guidelines

- [X] I have read the guidelines.

### Version/Commit hash

4df39adc

### Describe the bug.

Follow the directions listed in [BUILDING.md](https://git.sr.ht/~tenacity/tenacity/tree/master/BUILDING.md), with gcc 8.3.0. See the error due to a missing "copy_n" function in the standard c++ libraries.

### Expected behavior

I expected Tenacity to build without error so I could install it (since there do not appear to be any packaged releases).

### OS

MX Linux 19.4 / Devuan Beowulf 3.1

### Additional context

I installed MX Linux 19.4, then changed source repos to Devuan Beowulf 3.1, so gcc is from Devuan repos.
Since GitHub refuses to process the log, I uploaded it to [Lufi](https://upload.disroot.org/r/GTbY4MSf#XVcCrg6lbRm0sa6GUo4IYwswnPwQnNph5rsLtamtCyk=). 
Pass: impedingseminarnintendobackspinrecycleruninjured

### This issue is not a duplicate

- [X] I have confirmed this issue isn't a duplicate.

### PR DESCRIPTION
**Pull request by akleja**
Resolves: https://github.com/tenacityteam/tenacity/issues/662

Adds include for copy_n in AColor.cpp to fix compile issues on some systems

<details>
<summary>Checklist</summary>

- [x] I have signed off my commits using `-s` or `Signed-off-by`\* (See: [Contributing § DCO](https://github.com/tenacityteam/tenacity/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))
- [x] I made sure the code compiles on my machine
- [x] I made sure there are no unnecessary changes in the code\*
- [x] I made sure the title of the PR reflects the core meaning of the issue you are solving\*
- [x] I made sure the commit message(s) contain a description and answer the question "Why do those changes fix that particular issue?" or "Why are those changes really necessary as improvements?"\*

\* indicates required

</details>

### CODE REVIEW

**Review by n0toose — DISMISSED** [review]
Tested successfully.

**Comment by n0toose** [comment]
I'm considering about prematurely merging this because of this bug rendering compilation impossible in some systems (seems "urgent") and because of how minor it is. I'm really hesitant to proceed, even with the justification, so I'm going to let this run its course unless if I'm told otherwise.

File: src/AColor.cpp [hunk 1]
```diff
@@ -17,7 +17,7 @@ It is also a place to document colour usage policy in Audacity
 
 *//********************************************************************/
 
-
+#include <algorithm>
```
Comment (leio) [inline src/AColor.cpp]: The convention seems to be that std includes come after the matching class includes, so it should be added just above `<wx/window.h>`, not as the very first one

**Review by leio — DISMISSED** [review]

File: src/AColor.cpp (hunk 1 above)
Comment (leio) [reply src/AColor.cpp]: Also keep two empty lines between the file doc comments and first includes

Comment (akleja) [reply src/AColor.cpp]: Okay, got it!

**Comment by akleja** [comment]
Fixed formatting (and accidental mess up of commit message)

**Review by leio — APPROVED** [review]

**Review by n0toose — APPROVED** [review]

**Comment by n0toose** [comment]
Should've been a bit more proactive with the formatting, will keep that in mind.

**Pull request merged**